tailwind.config.js

```diff
@@ -9,16 +9,79 @@ module.exports = {
   darkMode: 'class',
   theme: {
     extend: {
-      fontFamily: {
-        serif: ['Source Sans Pro', ...defaultTheme.fontFamily.serif],
-        sans: ['Montserrat', ...defaultTheme.fontFamily.sans],
-        syne: ['Syne', ...defaultTheme.fontFamily.sans],
-      },
       colors: {
         primary: colors.teal,
         gray: colors.neutral,
+        'deth-pink': '#FF34F0',
+        'deth-purple': '#9B37FF',
+        'deth-error': '#FF3459',
+        'deth-success': '#4ADE80',
+        'deth-white': '#FFFFFF',
+        'deth-gray-100': '#F2EFF6',
```

Comment:
1. You could nest grays in an object.

https://tailwindcss.com/docs/customizing-colors#color-object-syntax

2. As this is entirely "deth"-themed, could we just skip `deth-` at the start of all of these?